### DISCUSSION TITLE
Add more log info for isApproved

### DISCUSSION THREAD

**Discussion by ssfxate** [Resnovas/smartcloud#224] [Q&A]
Hey,
I have two persons approved but the label isn't going to be added by the action.
![image](https://user-images.githubusercontent.com/18144023/133116688-00123c35-3a6b-4162-b203-0c8e0c243c31.png)
![image](https://user-images.githubusercontent.com/18144023/133116906-04acc08a-1ca9-4252-86ec-df54217adce3.png)

I have no idea how many approves has it counted already, who these persons were, or I did a mistake somewhere in the config…

Could you please add more info to this condition to log a list of persons that approved, list of all reviewers, current amount of approves?

**Comment by TGTGamer** [comment]
Heya, 

So the isApproved checks the number of approvals versus the number required, and that the pull request is not awaiting reviews. From the sounds of it, you are experiencing a bug. 

Would you be able to send me the link to your logs or the full context so I can test this issue offline?

**Reply by ssfxate** [reply]
Do you mean that the pull request should have no other awaiting reviews and required amount of approved to pass?
Example for **required 2**.
✅ Person 1
✅ Person 2
🟠 Person 3

Will it pass?

**Reply by TGTGamer** [reply]
No this would fail

**Reply by ssfxate** [reply]
In our project we should have **at least 2** approves, but requested reviewers could me more than 2.
2 isn't the max value. It's just the min value, but reviewers could be more, so approves could be more as well.

Have you got the same scheme (requirements) somewhere in a company/project?

I'm ready to send you the log. Would ur jonathan@videndum.studio be ok for this?

**Reply by TGTGamer** [reply]
Yeah, that email would be great. 

The way this was designed was under the assumption that users wouldn't want to mark the pull request as approved (label wise) until every user who had been requested to review has completed it. I understand the scheme you have explained and just overlooked it. 

I'm currently working on version 1.0.0-alpha.4 (jumping from the pre alpha 0.0.0-alpha.x) so will make this change in one of the upcoming developments - see #233

**Reply by ssfxate** [reply]
Hey, @TGTGamer have you got my email? I sent it 16 september.
To clarify this I'm not pushing you at all. I've just wanted to be sure, that my part is fulfilled.
